### ISSUE TITLE
Add Animal time_collected and treatment to load_samples command and SampleTableLoader class

### ISSUE THREAD

**Issue by jcmatese** [Princeton-LSI-ResearchComputing/tracebase#94]
## FEATURE DESCRIPTION

Modify sample loading command and class to deal with user-supplied "Collection Time" and "Animal Treatment"

### Feature Inspiration

Micheal supplied a [new dataset](https://drive.google.com/drive/u/0/folders/1v7QwchR1xXjdhgkgrvIGexhdG9LD8y6E), with additional columns/attributes for tracking.  This models have been updated #86 (or are pending #93) , but code does not currently use them.

### Feature Description

- add novel file to `DataRepo/example_data`
- load the attributes from the file, including the treatment protocol
- perhaps add additional command tests for this specific example file?

### Alternatives Considered

A brief description of any alternative features that may accomplish the same
ultimate goal.

### Comment

-----

### Assumptions

- We will assume input is correct

### Requirements

- List of conditions to be met for the feature
- E.g. Every column/row must display a value, i.e. cannot be empty

### Limitations

- A list of things this work will specifically not do
- E.g. This feature will only handle the most frequent use case X

### Affected/Changed Components

- DataRepo/utils.py (`SampleTableLoader`)
- DataRepo/management/commands/load_samples.py (perhaps)
- DataRepo/example_data/obob_sample_table_headers.yaml (**_or new yaml_**?)
- new DataRepo/example_data/ data added

#### GUI Change description

I don't think this should change any command interfaces, but opens up additional sample-annotations column options for submission, computation,  and eventual display on the web.

#### Code Change Description *(Pseudocode optional)*

add new columns to DataRepo/example_data/ yaml
deal with new columns in added data file
if a Animal Treatment is provided, then a protocol will be found or inserted

#### Tests

If feasible, a data load command might be added.

**Comment by jcmatese** [comment]
@lparsons @hepcat72 This is what I am proposing to do this afternoon, time permitting.  Lance, to you think it is preferable to add a new YAML, edit the existing YAML, or make a "generic" YAML?  I am uncertain on the restrictions/assumptions the yaml might introduce. [e.g.  if I add "Collection Time" to the yaml, and the file does not actually have that column, what is the outcome? (I have not tested, but thought you might know offhand)]

**Comment by lparsons** [comment]
By "YAML", I presume you mean the example [`obob_sample_table_headers.yaml`](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/DataRepo/example_data/obob_sample_table_headers.yaml). I would update the existing one, I don't see the point of multiple versions of that. Esp. since we're going to need the "time collected" for samples going forward.

**Comment by jcmatese** [comment]
Yes, that is what I meant.  I just wan't sure what the ramifications were.  If I add it to the [`obob_sample_table_headers.yaml`](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/DataRepo/example_data/obob_sample_table_headers.yaml) , would the cause preexisting files that lacked that column to file somehow, because treatment and time_collected are nullable in the model, and presumably could be nullable/absent in the file, too.

**Comment by jcmatese** [comment]
Seemed to run **existing** tests successfully if I just edit `obob_sample_table_headers.yaml` , `SampleTableHeaders` and `DefaultSampleTableHeaders` so I will work with that assumption.

**Comment by lparsons** [comment]
My suggestion would be to separate the sample time collection from the treatment stuff. There is already an issue for adding sample time collection to the loader and it should be independent of the new treatment variable. I might suggest that we add some sample data and tests to the PR that implements treatment protocols instead.

**Comment by jcmatese** [comment]
will remove time_collected from this issue, assuming it will be handled in #92
